liip/imagine-bundle/1.7/etc/routing/imagine.yaml

```diff
@@ -0,0 +1,19 @@
+liip_imagine_filter: 
+  path: "/media/cache/resolve/{filter}/{path}"
+  defaults: 
+    _controller: "%liip_imagine.controller.filter_action%"
+  methods: 
+    - GET
+  requirements: 
+    filter: "[A-z0-9_\\-]*"
+    path: .+
+
+liip_imagine_filter_runtime: 
+  path: "/media/cache/resolve/{filter}/rc/{hash}/{path}"
+  defaults: 
+    _controller: "%liip_imagine.controller.filter_runtime_action%"
+  methods: 
+    - GET
+  requirements: 
+    filter: "[A-z0-9_\\-]*"
+    path: .+
```

Comment:
Maybe use 4 spaces instead of 2 to keep consistency with the rest of the codebase?